stratosphere/pm/source/pm_main.cpp

```diff
@@ -145,7 +145,7 @@ int main(int argc, char **argv)
     s_server_manager.AddWaitable(new ServiceServer<ShellService>("pm:shell", 3));
     s_server_manager.AddWaitable(new ServiceServer<DebugMonitorService>("pm:dmnt", 3));
     s_server_manager.AddWaitable(new ServiceServer<BootModeService>("pm:bm", 6));
-    s_server_manager.AddWaitable(new ServiceServer<InformationService>("pm:info", 3));
+    s_server_manager.AddWaitable(new ServiceServer<InformationService>("pm:info", 19));
```

Comment:
add comment noting this difference from nintendo's limit and why it's been changed

Reply:
Yeah, will be done during pm rewrite.